.github/workflows/Linter.yml

```diff
@@ -68,13 +68,4 @@ jobs:
         run: |
          chmod +x style/run-lint.sh
          style/run-lint.sh shell
-  Comment:
-    needs: [Branch,C,Cppcheck,Python,Shell]
-    runs-on: ubuntu-latest
-    steps:
-      - name: Add Comment
-        uses: mshick/add-pr-comment@v1
-        with:
-          message: |
-            **All Lint Tests Passed!🚀**
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@bdevierno1 Just wondering why we're getting rid of this?

Reply:
When I made the PR it failed to comment. Job had status of failed. Was the first time that had happened but since it was more of a 'nice to have' I have opted to remove it. Probably would have posted if used 're-run jobs' but I think that the benefit of leaving it in is outweighed by the potential of it not working.